react-delegationdashboard/src/assets/styles/theme.scss

```diff
@@ -13,9 +13,11 @@ Override Bootstrap variables only above.
 
 // Main Shards Dashboard
 @import './shards-dashboard/shards-dashboards';
-@import './../../components/StatCard/stats.scss';
```

Comment:
I would have only one `modules` folder, and `stats.scss`, `trim.scss` etc inside